dashboard/test/controllers/api/v1/projects/section_projects_controller_test.rb

```diff
@@ -0,0 +1,55 @@
+require 'test_helper'
+
+class Api::V1::Projects::SectionProjectsControllerTest < ActionController::TestCase
+  STUDENT_STORAGE_ID = 11
+
+  setup do
+    @teacher = create :teacher
+    @section = create :section, user: @teacher
+    @student = create :user
+    create :follower, section: @section, student_user: @student
+
+    student_project_value = '{"name":"Bobs App","level":"/projects/applab","createdAt":"2017-01-24T16:41:08.000-08:00",'\
+      '"updatedAt":"2017-01-25T17:48:12.358-08:00"}'
+    student_project = {id: 22, value: student_project_value}
+
+    hidden_project_value = '{"name":"Hidden App","level":"/projects/playlab","createdAt":"2017-01-01T00:00:00.000-08:00",'\
+      '"updatedAt":"2017-01-01T00:00:00.000-08:00","hidden":true}'
```

Comment:
not super important, but since this is spanning multiple rows anyway it might be more readable to split over multiple lines. You can even leverage Ruby's hash/JSON support like this:

```ruby
hidden_project_value = {
  name: 'Hidden App',
  level: '/projects/playlab',
  createdAt: '2017-01-01T00:00:00.000-08:00',
  updatedAt: '2017-01-01T00:00:00.000-08:00',
  hidden: true
}.to_json
```